### PR TITLE
Handle functions as config values appropriately

### PR DESCRIPTION
**Pull request by herschel666**

### CODE REVIEW

**Comment by herschel666** [comment]
btw: we should imho put this rule ([`7e25605`](https://github.com/untool/untool/pull/560/commits/7e25605025e51a9e929b3b8898cf2dbbec9c6f5f)) into our eslint config.

File: packages/core/lib/utils.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ exports.placeholdify = (config) => {
         {}
       );
     }
-    if (regExp.test(item)) {
+    if (typeof item === 'string' && regExp.test(item)) {
```
Comment (KnisterPeter) [inline packages/core/lib/utils.js]: How about adding a test case for this?

Comment (herschel666) [reply packages/core/lib/utils.js]: Good idea!

Comment (herschel666) [reply packages/core/lib/utils.js]: Done.

**Pull request merged**